packages/app-runtime/src/modules/appEvents/RelationshipTemplateProcessedModule.ts

```diff
@@ -21,6 +21,7 @@ export class RelationshipTemplateProcessedModule extends AppRuntimeModule<Relati
 
         const data = event.data;
         switch (data.result) {
+            case RelationshipTemplateProcessedResult.RequestAutomaticallyDecided:
```

Comment:
Is this what we want here?

Reply:
no, in this case we want to hide the request from the user. So just noop here.